adodbapi/readme.txt

```diff
@@ -69,7 +69,7 @@ what's new in version 2.5
 
 License
 -------
-LGPL, see http://www.opensource.org/licenses/lgpl-license.php
+LGPL, see https://opensource.org/license/lgpl-2-1

```

Comment:
Important license link update ! The old link redirected to LGPL v3, which I'm pretty sure was incorrect given the current `license.txt` file

Reply:
the licenses are screwed up :(

Reply:
Cross-linking to https://github.com/mhammond/pywin32/issues/1127